### ISSUE TITLE
Solving vacuum equilibria from input file is somewhat broken

### ISSUE THREAD

**Issue by f0uriest** [PlasmaControl/DESC#278]
If an input file has `objective = vacuum` and no profiles given, the equilibrium is generated with a current profile. However, this is not constrained during optimization, and the resulting equilibrium can have a current profile with very large coefficients (~1e5). These seem to cancel to give a reasonable (if oscillatory) current profile, but it could cause issues with the optimizer since the step length its taking are O(1e4). 

Explicitly including `i = 0` in the input file seems to work, allowing iota to vary and giving reasonable step sizes. 

We should probably define what sort of behavior we want when `objective = vacuum`. Does this mean:
- Minimize J with I=0
- Minimize J with iota free
- something else?

**Comment by f0uriest** [comment]
For reference, after solving ``tests/inputs/HELIOTRON_vacuum`` I get the following current profile:
![image](https://user-images.githubusercontent.com/22567458/192936760-eab6028e-205f-48ec-adb3-eb1ca89155f0.png)
 
but these current profile coefficients:
```
array([-1.74327825e-01,  8.02535939e+02, -1.49727649e+04,  1.01389452e+05,
       -3.22095663e+05,  5.18709088e+05, -4.09661802e+05,  1.25878001e+05])
```

**Comment by dpanici** [comment]
I feel like true "vacuum" is minimize J with iota free, as J=0 is vacuum, but most people in the community think of minimizing energy (or force) subject to I=0 current constraint as "vacuum"

Those results look pretty different though, and the iota near axis being off is not good either.

**Comment by f0uriest** [comment]
One possible fix to this would be related to #275, so we can always include the profile constraints even if they're not needed. So we'd assume pressure and current are always fixed, and iota is fixed unless `objective=vacuum`

**Comment by f0uriest** [comment]
FYI the weird profiles above were due to plotting using a linear grid. When taking the averages properly we get the same iota profile and similar current profiles
